src/pages/library/DataBrowserPreview.js

```diff
@@ -83,7 +83,8 @@ const DataBrowserPreview = ({ id }) => {
           style: { fontSize: 16, textTransform: 'none' },
           onClick: () => setViewJSON({ title: `${table}, Row ${rowIndex} - ${cellKey}`, cellData: maybeJSON })
         }, ['View JSON'])],
-        [Utils.DEFAULT, () => cellContent]
+        [typeof cellContent === 'object', () => JSON.stringify(cellContent)],
```

Comment:
Any reason not to use `_.isObject`? It seems much friendlier.

Reply:
Nope, that seems better to me!